aviatrix/resource_aviatrix_spoke_gateway.go

```diff
@@ -174,6 +174,7 @@ func resourceAviatrixSpokeGateway() *schema.Resource {
 				Type:        schema.TypeBool,
 				Optional:    true,
 				Default:     false,
+				Deprecated:  "Non ActiveMesh features will be completely deprecated and removed in provider 2.21.0+. Here is the guide for Migrating from Classic Aviatrix Encrypted Transit Network to Aviatrix ActiveMesh Transit Network: https://registry.terraform.io/providers/AviatrixSystems/aviatrix/latest/docs/guides/migrating_to_active_mesh_transit_network",
```

Comment:
Non-ActiveMesh .... Please see the following guide to migrate from Classic Aviatrix Encrypted Transit Network to Aviatrix ActiveMesh Transit Network: < link >